app/src/main/java/com/osfans/trime/ime/symbol/LiquidLayout.kt

```diff
@@ -65,13 +72,7 @@ class LiquidLayout(context: Context, service: TrimeInputMethodService, theme: Th
                             }
                         val root =
                             frameLayout {
-                                background =
-                                    GradientDrawable().apply {
-                                        cornerRadius = theme.generalStyle.roundCorner.toFloat()
-                                        ColorManager.getColor("key_back_color")?.let { bg ->
-                                            setColor(bg)
-                                        }
-                                    }
+                                background = mBackground
```

Comment:
Since it's only used once, may be passing the method as the right value directly is better.